src/containers/tud/TimeUseDiaryContainer.js

```diff
@@ -8,18 +8,24 @@ import { DataProcessingUtils, Paged } from 'lattice-fabricate';
 import {
   AppContainerWrapper,
   AppContentWrapper,
-  AppHeaderWrapper,
   Card,
   CardSegment,
   Typography
 } from 'lattice-ui-kit';
 import { useRequestState } from 'lattice-utils';
+// $FlowFixMe
+import { useTranslation } from 'react-i18next';
```

Comment:
The `react-i18next` flow-typed definition is missing both `useTranslation` and `initReactI18next`, so it's useless to us. I recommend uninstalling it and just creating a stub so that you can get rid of the `$FlowFixMe` in the code.